support/cas-server-support-pm-webflow/src/main/java/org/apereo/cas/pm/web/flow/actions/SendPasswordResetInstructionsAction.java

```diff
@@ -25,7 +25,7 @@
  */
 public class SendPasswordResetInstructionsAction extends AbstractAction {
     /** Param name for the token. */
-    public static final String PARAMETER_NAME_TOKEN = "t";
+    public static final String PARAMETER_NAME_TOKEN = "pswdrst";
```

Comment:
No objections, but why the name change?

Reply:
I almost left it, but when it was /pwsrst?t=<token> it was easy to see what was going on. But now that it's a subflow having it be /login?t=<token> was less clear, so I opted to change it so that even from the URL it's clear what's going on. I'm good either way, though.